Application/Commands/Enquiry/Respond/DeclineEnquiryResponseCommand.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Application.Common.Interfaces;
+using Application.Common.Models.Enquiry.Respond;
+using Microsoft.Extensions.Logging;
+
+namespace Application.Commands.Enquiry.Respond;
+
+public record DeclineEnquiryResponseCommand(
+    string SupportReferenceNumber,
+    string TuitionPartnerSeoUrl) : IRequest<bool>
+{ }
+
+public class DeclineEnquiryResponseCommandHandler : IRequestHandler<DeclineEnquiryResponseCommand, bool>
+{
+    private readonly IUnitOfWork _unitOfWork;
+
+    private readonly ILogger<DeclineEnquiryResponseCommandHandler> _logger;
+
+    public DeclineEnquiryResponseCommandHandler(IUnitOfWork unitOfWork,
+        ILogger<DeclineEnquiryResponseCommandHandler> logger)
+    {
+        _unitOfWork = unitOfWork;
+        _logger = logger;
+    }
+
+    public async Task<bool> Handle(DeclineEnquiryResponseCommand request, CancellationToken cancellationToken)
+    {
+        var result = new ResponseConfirmationModel();
```

Comment:
Can we remove the unused local variable `result`?

Reply:
Done